sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -72,58 +72,64 @@ case class CurrentTimestamp() extends LeafExpression with CodegenFallback {
 }
 
 /**
- * Adds a number of days to startdate.
+ * The base for addition/subtraction for days.
```

Comment:
It seems a bit messy here. I made a common parent for `AddDays` and `SubDays` named `AddDaysBase`.